### ISSUE TITLE
novelall.com - html code in command prompt

### ISSUE THREAD

**Issue by SirGryphin** [dipu-bd/lightnovel-crawler#80]
When downloading book from novelall.com after it says **Retrieving novel info...** I get all the html code, here is some of what I'm seeing.

I can download and create a epub without any problems though just it list all the chapters in html before asking for download directory.

```
[<a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-1/1262253/" title="Godly Empress Doctor Chapter 1">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-2/1278895/" title="Godly Empress Doctor Chapter 2">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-3/1284268/" title="Godly Empress Doctor Chapter 3">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-4/1284281/" title="Godly Empress Doctor Chapter 4">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-5/1289918/" title="Godly Empress Doctor Chapter 5">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-6/1307369/" title="Godly Empress Doctor Chapter 6">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-7/1307532/" title="Godly Empress Doctor Chapter 7">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-8/1317611/" title="Godly Empress Doctor Chapter 8">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-9/1317612/" title="Godly Empress Doctor Chapter 9">

</a>, <a href="https://www.novelall.com/chapter/Godly-Empress-Doctor-Chapter-10/1336914/" title="Godly Empress Doctor Chapter 10">
```

- Novel URL: https://www.novelall.com/novel/Godly-Empress-Doctor.html
- App Version: 2.7.9
- Package: pip
- OS: Windows 10

**Comment by yudilee** [comment]
My bad, i leave command to print chapter link url on line 62 in novelall crawler. i will pull request on this fix

**Comment by dipu-bd** [comment]
the fix will be available in 2.7.10
